content/project-news/hermes-belusca-hired-full-time.md

```diff
@@ -0,0 +1,27 @@
+---
+title:       "Hermès BÉLUSCA - MAÏTO hired full-time to work on the ReactOS GUI Setup"
+author:      "GeoB99"
+date:        2023-09-28
+---
+
+Our team proudly announces that ReactOS Deutschland e.V. has hired Hermès BÉLUSCA - MAÏTO to work full-time on the ReactOS GUI Setup for the next 3 months.
+
+Hermès is a long-time contributor and developer who worked on the project since 2012.
+He has several skills and expertise into various components of ReactOS, notably being the Client/Server Runtime Subsystem (CSRSS), Console Server Subsystem, NTVDM, and others.
+
+His upcoming goal is to finish the first-stage ReactOS GUI setup, which offers an alternative to the classic first-stage text-mode setup (also called USETUP).
+Currently the only option to install ReactOS is through the text-mode setup, with the rest of the setup installation being handled after the second-stage setup.
+This has been the standard method of installing ReactOS on virtual machines or bare metal since the inception of the project.
+
+His roadmap for the GUI Setup is as follows:
+
+* Finish partial-Winesync of setupapi.dll regarding CAB-file extraction;
+* Move FreeLdr installation choice step after the ROS disk/partition-choice step and **JUST** before the actual installation;
+* Integrate registry settings callbacks (currently absent in the GUI setup);
+* Add GPT support for the GUI Setup;
+* Miscellaneous stuff (further cleanup, some refactorings, etc).
+
+More information can be found [here](https://reactos.org/wiki/First_Stage_GUI_Setup).
+
+With the help of donations from the community, we were able to finance his full-time paid contract.
+The ReactOS team is very thankful for your donations!
```

Comment:
Maybe add something here like "Stay tuned, the progress of this work will be followed by the blog posts by Hermès".